exclude.yml

```
@@ -0,0 +1,2 @@
+- sub-1005491_T2w.nii.gz # Don't see C2-3 disc, image tilted
```

to follow [E261](https://www.flake8rules.com/rules/E261.html)
```suggestion
- sub-1005491_T2w.nii.gz  # Don't see C2-3 disc, image tilted
```